frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { FaInfo } from 'react-icons/fa';
+import Control from 'react-leaflet-control';
+import styled from 'styled-components';
+import clsx from 'classnames';
+import * as L from 'leaflet';
+import { useEffect } from 'react';
+import { PropertyPopUpContext } from '../Map';
+import HeaderActions from './HeaderActions';
+import { InfoContent } from './InfoContent';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+
+const InfoContainer = styled.div`
+  margin-right: -10px;
```

Comment:
What is this negative margin being used for?

Reply:
it keeps the container against the edge of the screen, it floats on the map otherwise